UPGRADE-14.0.md

```diff
@@ -190,4 +190,8 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 -   fix Comparison for not logged in users ([#2905](https://github.com/shopsys/shopsys/pull/2905))
     -   unified code for Comparison and Wishlist
     -   refactored Zustand store to use only one store (User Store) for all cartUuid, wishlistUuid and comparisonUuid
+-   added skeletons for Wishlist, Comparison and My orders, Order detail and Product Variants page ([#2906](https://github.com/shopsys/shopsys/pull/2906))
+    -   see #project-base-diff to update your project
+-   fix Add to cart/wishlist/comparison feature when using multiple tabs ([#2906](https://github.com/shopsys/shopsys/pull/2906))
+    -   As fresh unlogged user without `cartUuid` I open multiple tabs, then in each tab I add product to cart, I am experiencing that each tab contains different products in cart, even after refresh. Same applies to `Wishlist` and `Comparison`. This was solved by adding `BroadcastChannel` middleware to Zustand store, so now it uses always only first returned `uuid`.
```

Comment:
I would use a different formulation. Something like "There was an error". Now when a person reads this, he reads a problem description, without knowing this is a past, solved error.

Reply:
I don't see this so understandable but I used better formulation.